// pdf-parse/dist/cjs/geometry/Geometry.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Table = exports.Shape = exports.Rectangle = exports.Point = exports.LineStore = exports.LineDirection = exports.Line = void 0;
var Line_1 = require("./Line");
Object.defineProperty(exports, "Line", { enumerable: true, get: function () { return Line_1.Line; } });
Object.defineProperty(exports, "LineDirection", { enumerable: true, get: function () { return Line_1.LineDirection; } });
var LineStore_1 = require("./LineStore");
Object.defineProperty(exports, "LineStore", { enumerable: true, get: function () { return LineStore_1.LineStore; } });
var Point_1 = require("./Point");
Object.defineProperty(exports, "Point", { enumerable: true, get: function () { return Point_1.Point; } });
var Rectangle_1 = require("./Rectangle");
Object.defineProperty(exports, "Rectangle", { enumerable: true, get: function () { return Rectangle_1.Rectangle; } });
var Shape_1 = require("./Shape");
Object.defineProperty(exports, "Shape", { enumerable: true, get: function () { return Shape_1.Shape; } });
var Table_1 = require("./Table");
Object.defineProperty(exports, "Table", { enumerable: true, get: function () { return Table_1.Table; } });
